src/module.c

```diff
@@ -13532,6 +13535,111 @@ int RM_GetDbIdFromDefragCtx(RedisModuleDefragCtx *ctx) {
     return ctx->dbid;
 }
 
+/* --------------------------------------------------------------------------
```

Comment:
There is a strong desire for me to move the implementation of this whole piece behind `RedisModule_StreamTrimByID`. 😁

Reply:
Thank you :-)